binding.gyp

```diff
@@ -0,0 +1,19 @@
+{
+  "targets": [
+    {
+      "target_name": "erlpack",
+      "sources": [ "js/erlpack.cc" ],
+      "include_dirs": [
+        "<!(node -e \"require('nan')\")",
+        "electron/vendor/brightray/vendor/download/libchromiumcontent/src"
+      ],
+      'conditions': [
+        [ 'OS=="mac"', {
```

Comment:
nope. c++ features are exposed based on the version of msvs -- afaik you cannot flag it to support older / newer standards (you can just choose "c" or "c++".)

fwiw, though, this might be better expressed with cflags_cc which gyp generators should map to their ide equivalents (i.e. the xcode setting is actually ```CLANG_CXX_LANGUAGE_STANDARD```):
```
'cflags_cc': ['-std=c++11'],
```